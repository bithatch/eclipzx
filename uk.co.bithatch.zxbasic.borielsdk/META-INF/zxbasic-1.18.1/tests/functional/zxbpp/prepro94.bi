

#ifdef MACRO
#  ifndef ANOTHER_MACRO
#    error This should not happen
#  endif
#else
  OK
#endif
