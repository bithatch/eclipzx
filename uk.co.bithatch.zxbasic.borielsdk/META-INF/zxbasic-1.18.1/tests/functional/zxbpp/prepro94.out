#line 1 "prepro94.bi"


#line 8 "prepro94.bi"
  OK
#line 10 "prepro94.bi"
